slippi/event.py

```diff
@@ -225,10 +225,12 @@ class End(Base):
 
     method: End.Method #: `changed(2.0.0)` How the game ended
     lras_initiator: Optional[int] #: `added(2.0.0)` Index of player that LRAS'd, if any
+    player_placements: Optional[List[int]] #: `added (3.13.0)` 0-indexed placement positions. -1 if player not in game
 
-    def __init__(self, method: End.Method, lras_initiator: Optional[int] = None):
+    def __init__(self, method: End.Method, lras_initiator: Optional[int], player_placements: Optional[List[int]] = None):
```

Comment:
Looks like tests are failing since the `lras_initiator` default was removed:

```suggestion
    def __init__(self, method: End.Method, lras_initiator: Optional[int] = None, player_placements: Optional[List[int]] = None):
```

Reply:
oof my bad, fixed the issue. I'm failing another test but it's also failing on the master branch so it might be related to my environment